packages/flutter/lib/src/widgets/gesture_detector.dart

```diff
@@ -804,23 +806,49 @@ class RawGestureDetector extends StatefulWidget {
   /// duplication of information.
   final bool excludeFromSemantics;
 
+  /// Describes the semantics notations that should be added to the underlying
+  /// render object [RenderSemanticsGestureHandler].
+  ///
+  /// It has no effect if [excludeFromSemantics] is true.
+  ///
+  /// When [semantics] is null, [RawGestureDetector] will fall back to a
+  /// default delegate that behaves as follows:
+  ///
+  ///  * During a semantic tap, it looks for [TapGestureRecognizer] and calls
```

Comment:
Can you specify in the docs where "it looks"?

Reply:
Fixed